// @paralleldrive/react-feature-toggles/dist/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _configureFeature = require('./configure-feature');

Object.defineProperty(exports, 'configureFeature', {
  enumerable: true,
  get: function get() {
    return _configureFeature.configureFeature;
  }
});

var _featureToggles = require('./feature-toggles');

Object.defineProperty(exports, 'FeatureToggles', {
  enumerable: true,
  get: function get() {
    return _featureToggles.FeatureToggles;
  }
});

var _feature = require('./feature');

Object.defineProperty(exports, 'Feature', {
  enumerable: true,
  get: function get() {
    return _feature.Feature;
  }
});

var _withFeatureToggles = require('./with-feature-toggles');

Object.defineProperty(exports, 'withFeatureToggles', {
  enumerable: true,
  get: function get() {
    return _withFeatureToggles.withFeatureToggles;
  }
});

var _context = require('./context');

Object.defineProperty(exports, 'FeatureTogglesContext', {
  enumerable: true,
  get: function get() {
    return _context.FeatureTogglesContext;
  }
});

var _withFeatures = require('./with-features');

Object.defineProperty(exports, 'withFeatures', {
  enumerable: true,
  get: function get() {
    return _withFeatures.withFeatures;
  }
});

var _useFeatures = require('./use-features');

Object.defineProperty(exports, 'useFeatures', {
  enumerable: true,
  get: function get() {
    return _useFeatures.useFeatures;
  }
});
