// @paralleldrive/feature-toggles/dist/index.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _activateFeatures = require('./activate-features');

Object.defineProperty(exports, 'activateFeatures', {
  enumerable: true,
  get: function get() {
    return _activateFeatures.activateFeatures;
  }
});

var _getActiveFeatureNames = require('./get-active-feature-names');

Object.defineProperty(exports, 'getActiveFeatureNames', {
  enumerable: true,
  get: function get() {
    return _getActiveFeatureNames.getActiveFeatureNames;
  }
});

var _mergeFeatureNames = require('./merge-feature-names');

Object.defineProperty(exports, 'mergeFeatureNames', {
  enumerable: true,
  get: function get() {
    return _mergeFeatureNames.mergeFeatureNames;
  }
});

var _isActiveFeatureName = require('./is-active-feature-name');

Object.defineProperty(exports, 'isActiveFeatureName', {
  enumerable: true,
  get: function get() {
    return _isActiveFeatureName.isActiveFeatureName;
  }
});

var _getQueryFeatureNames = require('./get-query-feature-names');

Object.defineProperty(exports, 'getQueryFeatureNames', {
  enumerable: true,
  get: function get() {
    return _getQueryFeatureNames.getQueryFeatureNames;
  }
});

var _removeFeatureNames = require('./remove-feature-names');

Object.defineProperty(exports, 'removeFeatureNames', {
  enumerable: true,
  get: function get() {
    return _removeFeatureNames.removeFeatureNames;
  }
});

var _getReqQueryFeatureNames = require('./get-req-query-feature-names');

Object.defineProperty(exports, 'getReqQueryFeatureNames', {
  enumerable: true,
  get: function get() {
    return _getReqQueryFeatureNames.getReqQueryFeatureNames;
  }
});

var _getCurrentActiveFeatureNames = require('./get-current-active-feature-names');

Object.defineProperty(exports, 'getCurrentActiveFeatureNames', {
  enumerable: true,
  get: function get() {
    return _getCurrentActiveFeatureNames.getCurrentActiveFeatureNames;
  }
});

var _getBrowserQueryFeatureNames = require('./get-browser-query-feature-names');

Object.defineProperty(exports, 'getBrowserQueryFeatureNames', {
  enumerable: true,
  get: function get() {
    return _getBrowserQueryFeatureNames.getBrowserQueryFeatureNames;
  }
});

var _createExpressMiddleware = require('./create-express-middleware');

Object.defineProperty(exports, 'createExpressMiddleware', {
  enumerable: true,
  get: function get() {
    return _createExpressMiddleware.createExpressMiddleware;
  }
});
